test/integration/targets/azure_rm_virtualmachine_2/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/azure
+shippable/azure/group4
+destructive
+azure_rm_virtualmachine
```

Comment:
Creating a second target for the module isn't going to work correctly. Why not add to the existing test?

Reply:
oh, thanks for this comment.
I was just wondering how to do that...
extisting test is pretty time-consuming already.
anyway, I will try

Reply:
What parts of the existing test are slow? Perhaps it would benefit from being parallelized -- assuming there are at least two sequences of tasks that can safely run independently of each other.

Reply:
ok, removed additional tests...
i was also thinking about the future. there are some scenarios which involve 2 resources.
for instance virtual machine and managed disk.
in such case the test will be most likely in azure_rm_managed_disk.
i think in such case changes to the azure_rm_virtualmachine should trigger both tests.

Reply:
@mattclay could you approve now?